recipes/aws-c-cal/all/test_package/conanfile.py

```diff
@@ -13,5 +13,9 @@ def build(self):
 
     def test(self):
         if not tools.cross_building(self.settings):
+            stream = io.StringIO()
             bin_path = os.path.join("bin", "test_package")
-            self.run(bin_path, run_environment=True)
+            self.run(bin_path, run_environment=True, output=stream)
+            print(stream.getvalue())
+            if self.deps_user_info["aws-c-cal"].with_openssl == "True":
+                assert "found static libcrypto" in stream.getvalue()
```

Comment:
check for `found dynamic libcrypto` otherwise. maybe also check the version.

Reply:
@SSE4 
Sorry for late.

`found dynamic libcrypto` is not our case.
In this recipe, test program is always linked with libcrypto either static or dynamic.

In aws-c-cal, `found dynamic libcrypto` means that program was not linked by static or dynamic library and  try to find libcrypto library to dlopen it.
`found dynamic libcrypto` is not our case.
In this recipe, test program is always linked with libcrypto either static or dynamic.

In aws-c-cal, `found dynamic libcrypto` means that program was not linked by static or dynamic library and  try to find libcrypto library to dlopen it.